ERROR - 2024-12-19 05:42:48 --> ErrorException: pg_query(): Query failed: ERROR:  null value in column &quot;student_id&quot; of relation &quot;users&quot; violates not-null constraint
DETAIL:  Failing row contains (14, kai, $2y$10$RwTmWEYkhSbbzrv7Ytn8iOgIGXjUDzU1JK79vtTO4p9HYs9QQkbBq, 2024-12-19 13:42:48.709577, null). in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(98): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 05:45:58 --> ErrorException: pg_query(): Query failed: ERROR:  null value in column &quot;student_id&quot; of relation &quot;users&quot; violates not-null constraint
DETAIL:  Failing row contains (15, kai, $2y$10$F9xvw2WvZA2XbTyWAfvrhesb6qZbTMvr0xZgFZhyhPoVIn94LKjlu, 2024-12-19 13:45:58.74976, null). in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(101): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 05:51:48 --> ErrorException: pg_query(): Query failed: ERROR:  null value in column &quot;student_id&quot; of relation &quot;users&quot; violates not-null constraint
DETAIL:  Failing row contains (1, qin, $2y$10$z9SsnVDQ.KMSBHRy9wVvZ.8lh3iKDZiGAM7t64AImflIBOtyjyhXu, 2024-12-19 13:51:48.380606, null). in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(101): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 06:55:12 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;users&quot; does not exist
LINE 1: ... &quot;users&quot; (&quot;student_id&quot;, &quot;password&quot;, &quot;created_at&quot;, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(91): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 07:01:27 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;users&quot; does not exist
LINE 1: ... &quot;users&quot; (&quot;student_id&quot;, &quot;password&quot;, &quot;created_at&quot;, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(90): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 07:09:26 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;users&quot; does not exist
LINE 1: ... &quot;users&quot; (&quot;student_id&quot;, &quot;password&quot;, &quot;created_at&quot;, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(92): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 07:23:24 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;users&quot; does not exist
LINE 1: ... &quot;users&quot; (&quot;student_id&quot;, &quot;password&quot;, &quot;created_at&quot;, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(84): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 07:33:16 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;users&quot; does not exist
LINE 1: ... &quot;users&quot; (&quot;student_id&quot;, &quot;password&quot;, &quot;created_at&quot;, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(92): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:01:27 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: ...files&quot; (&quot;student_id&quot;, &quot;name&quot;, &quot;email&quot;, &quot;address&quot;, &quot;created_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(109): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:01:31 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(44): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
CRITICAL - 2024-12-19 14:01:31 --> ErrorException: Undefined variable $student
[Method: GET, Route: student-profile/edit/D20211097851]
in APPPATH\Views\sections\student_profile_edit.php on line 19.
 1 APPPATH\Views\sections\student_profile_edit.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $student', 'C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php', 19)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('sections/student_profile_edit', [], true)
 5 APPPATH\Controllers\StudentProfileController.php(48): view('sections/student_profile_edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\UPSI_SkillHub\\public\\index.php')
ERROR - 2024-12-19 14:01:47 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(44): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
CRITICAL - 2024-12-19 14:01:47 --> ErrorException: Undefined variable $student
[Method: GET, Route: student-profile/edit/D20211097851]
in APPPATH\Views\sections\student_profile_edit.php on line 19.
 1 APPPATH\Views\sections\student_profile_edit.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $student', 'C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php', 19)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('sections/student_profile_edit', [], true)
 5 APPPATH\Controllers\StudentProfileController.php(48): view('sections/student_profile_edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\UPSI_SkillHub\\public\\index.php')
ERROR - 2024-12-19 14:09:18 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(44): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
CRITICAL - 2024-12-19 14:09:18 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: student-profile/edit/D20211097851]
in APPPATH\Views\sections\student_profile_edit.php on line 19.
 1 APPPATH\Views\sections\student_profile_edit.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php', 19)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\UPSI_SkillHub\\app\\Views\\sections\\student_profile_edit.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('sections/student_profile_edit', [], true)
 5 APPPATH\Controllers\StudentProfileController.php(48): view('sections/student_profile_edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\UPSI_SkillHub\\public\\index.php')
ERROR - 2024-12-19 14:18:49 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:19:08 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:19:47 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:19:56 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:20:12 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:23:49 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;updated_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: ... &quot;city&quot; = NULL, &quot;state&quot; = NULL, &quot;country&quot; = NULL, &quot;updated_a...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(80): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:23:49 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:03 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:17 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:26 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:43 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(80): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:43 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:47 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 14:24:50 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 15:22:29 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 15:22:45 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(79): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 15:22:45 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 15:34:04 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(79): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 15:34:04 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097851')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:06 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(106): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:09 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097852')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:17 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(79): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:17 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097852')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:31 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(79): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:16:31 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097852')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:20:28 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(97): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:20:31 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097853')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:20:41 --> ErrorException: pg_query(): Query failed: ERROR:  invalid input syntax for type numeric: &quot;&quot;
LINE 1: ...status&quot; = '', &quot;ic_number&quot; = '', &quot;family_income&quot; = '', &quot;sibli...
                                                             ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'UPDATE "student...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "student...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "student...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE "student...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(79): CodeIgniter\Model->update()
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->update()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:20:42 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\StudentProfileController.php(43): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\StudentProfileController->edit('D20211097853')
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\StudentProfileController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 18:27:34 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(94): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
ERROR - 2024-12-19 19:13:16 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;created_at&quot; of relation &quot;student_profiles&quot; does not exist
LINE 1: INSERT INTO &quot;student_profiles&quot; (&quot;student_id&quot;, &quot;created_at&quot;, ...
                                                      ^ in C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php:204
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\laragon\\www\\...', 204)
#1 C:\laragon\www\UPSI_SkillHub\system\Database\Postgre\Connection.php(204): pg_query(Object(PgSql\Connection), 'INSERT INTO "st...')
#2 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(734): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "st...')
#3 C:\laragon\www\UPSI_SkillHub\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "st...')
#4 C:\laragon\www\UPSI_SkillHub\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO "st...', Array, false)
#5 C:\laragon\www\UPSI_SkillHub\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\laragon\www\UPSI_SkillHub\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#7 C:\laragon\www\UPSI_SkillHub\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\UPSI_SkillHub\app\Controllers\AuthController.php(94): CodeIgniter\Model->insert(Array)
#9 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(933): App\Controllers\AuthController->store()
#10 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\laragon\www\UPSI_SkillHub\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\UPSI_SkillHub\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\UPSI_SkillHub\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\UPSI_SkillHub\public\index.php(46): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\UPSI_SkillHub\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}
